solr/core/src/java/org/apache/solr/handler/admin/ContainerPluginsApi.java

```diff
@@ -64,15 +64,15 @@ public ContainerPluginsApi(CoreContainer coreContainer) {
 
   public class Read {
     @EndPoint(method = METHOD.GET,
-        path = "/cluster/plugin",
+        path = "/cluster/plugins",
```

Comment:
you know this is a backward incompatible change , right?

Reply:
Right, but this is for 9.0 so we can break back-compat if it's justified - and I think it is because the singular name here doesn't make sense, as it is a location where multiple plugin configurations are defined.

In any case we can provide a back-compat shim for 9.0 to also accept `plugin` singular.

Reply:
We can , but that doesn't mean we should. This is just a cosmetic change. Adds no value to user but causes enormous pain to the user. We just make people not upgrade at all

I'm -1 on doing a breaking change